packages/permissions/src/components/authorized/authorized.tsx

```diff
@@ -19,19 +20,38 @@ type TDemandedActionRight = {
   group: TActionRightGroup;
   name: TActionRightName;
 };
+type TDemandedDataFence = {
+  group: string;
+  name: string;
+  type: string;
+};
 type TActionRight = {
   [key: string]: boolean;
 };
 type TActionRights = {
   [key: string]: TActionRight;
 };
-
+type TDataFence = {
+  [key: string]: {};
+};
+type TResourceToApplyDataFence = {
+  id: string;
+  // Orders
+  storeRef?: {
```

Comment:
Maybe I'm missing something but why do we need to hardcode those things?

Reply:
How are these hardcoded?